custom_components/powercalc/data/osram/AB401130055/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plus 1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.9.8:docker"
+    },
+    "name": "LIGHTIFY Surface Light LED Tunable White (AB401130055)",
```

Id may be removed from the name. Name is only used in supported model listing. And this would be superfluous.

```suggestion
    "name": "LIGHTIFY Surface Light LED Tunable White",
```